src/steps/fetch-content.js

```diff
@@ -24,7 +24,11 @@ export default async function fetchContent(state, params, opts) {
     throw new Error('\'origin\' not set in converter.yaml');
   }
 
-  const mappedPath = mapInbound(path, mappingCfg || { mappings: ['/:/'] });
+  let mappedPath = mapInbound(path, mappingCfg || { mappings: ['/:/'] });
+  // if the mapped path has no extension, add .html
+  if (!mappedPath.includes('.') && !mappedPath.endsWith('/')) {
```

Comment:
the suffix should be configured in the converter configuration file and appended only when set

Reply:
I added it in https://github.com/Buuhuu/crosswalk-converter/pull/3/commits/01865f56a9a09aceb3683ff01a3d7c09b9abc642